CMakeLists.txt

```diff
@@ -1,52 +1,48 @@
-project(obs-websocket VERSION 5.1.0)
-set(OBS_WEBSOCKET_RPC_VERSION 1)
+cmake_minimum_required(VERSION 3.16...3.25)
 
-option(ENABLE_WEBSOCKET "Enable building OBS with websocket plugin" ON)
+if(NOT CMAKE_PROJECT_NAME)
+  project(obs-websocket VERSION 5.0.1)
```

Comment:
This looks like an old change?

Reply:
It's actually a fallback for building standalone because I wasn't aware that it's not a valid option anymore - that branch will be remove entirely.